maintenance/wikia/cronjobs/fetchOptimizelyScript.php

```diff
@@ -9,9 +9,9 @@
 
 require_once( dirname( __FILE__ ) . '/../../commandLine.inc' );
 
-global $wgDevelEnvironment, $wgOptimizelyUrl, $wgOptimizelyDevUrl;
+global $wgOptimizelyUrl, $wgOptimizelyDevUrl;
 
-$url = $wgDevelEnvironment ? $wgOptimizelyDevUrl : $wgOptimizelyUrl;
+$url = Optimizely::isOptimizelyDevEnv() ? $wgOptimizelyDevUrl : $wgOptimizelyUrl;
```

Comment:
Since you're already making a call to the Optimizely class, and since this ternary statement is in two places now, maybe the ternary could be part of an ```Optimizely::getOptimizelyUrl``` method?